packages/react-devtools-inline/src/frontend.js

```diff
@@ -16,7 +16,6 @@ export function createStore(bridge: FrontendBridge, config?: Config): Store {
   return new Store(bridge, {
     checkBridgeProtocolCompatibility: true,
     supportsTraceUpdates: true,
-    supportsTimeline: true,
```

Comment:
Is this intended?

Reply:
Yes, it was intended, but let me revert this to how it was previously implemented.